src/activity_item_handling.cpp

```diff
@@ -394,6 +395,63 @@ void activity_handlers::drop_do_turn( player_activity *act, player *p )
     put_into_vehicle_or_drop( *p, obtain_activity_items( *act, *p ), pos );
 }
 
+void activity_handlers::washing_finish( player_activity *act, player *p )
+{    
+    const std::string cur_time = to_string_time_of_day( calendar::turn );
+
+    auto items = reorder_for_dropping( *p, convert_to_indexes( *act ) );
+
+    // Check again that we have enough water and soap incase the amount in our inventory changed somehow
+    // Consume the water and soap
+    int required_water = 0;
+    int required_cleanser = 0;
+
+    std::list<act_item>::iterator item_to_clean;
+        for (item_to_clean = items.begin(); item_to_clean != items.end(); ++item_to_clean){
```

Comment:
Needs astyling here and below.

Reply:
I ran astyle on all of the files I changed, this ended up modifying some code I didn't work on. Is that OK, or should I specifically only astyle my code?